### PR TITLE
Change OMERO.server systemd nofile limit (IDR-0.3.2)

### PR DESCRIPTION
**Pull request by manics**
Systemd ignores the user ulimits, so this has to be modified using a systemd service configuration parameter.

https://trello.com/c/Kv8KjOvz/54-systemd-omero-ulimit

Before:
```
$ pgrep icegridnode
2287
$ grep files /proc/2287/limits
Max open files            1024                 4096                 files
```
After:
```
$ pgrep icegridnode
24753
$ grep files /proc/24753/limits
Max open files            16384                16384                files
```

### CODE REVIEW

**Comment by manics** [comment]
Updated to use `molecule`

**Comment by joshmoore** [comment]
> Travis needs be activated on this repo (and maybe others too?)

Good catch. The new API creation strategy rather than transfer requires that extra step (so perhaps the balance goes back to transfer?). All `ansible-role-*` repos have been activated. Re-opening this to trigger.

File: tests/requirements.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+- src: openmicroscopy.basedeps
```
Comment (joshmoore) [inline tests/requirements.yml]: Bit unfortunate that this has to be reproduced.

**Comment by joshmoore** [comment]
```
============================ pytest-warning summary ============================
WP1 None Module already imported so can not be re-written: testinfra
================= 10 passed, 1 pytest-warnings in 5.04 seconds =================
--> Destroying instances...
Stopping container omero-server-ice35...
Removed container omero-server-ice35.
Stopping container omero-server-ice36...
Removed container omero-server-ice36.
The command "molecule test" exited with 0.
```

Nice. Proposed version number? 1.1.0 since it's an addition of variables?

**Comment by manics** [comment]
`1.1.0` sounds reasonable

**Comment by joshmoore** [comment]
Pushed as https://github.com/openmicroscopy/ansible-role-omero-server/releases/tag/0.1.1

**Comment by joshmoore** [comment]
@manics pointed out the wrong tag - sorry. That tag has since been deleted and replaced by 1.1.0. The build on galaxy is unfortunately still present.

**Pull request merged**